rpcs3/Emu/system_config.h

```diff
@@ -250,6 +250,7 @@ struct cfg_root : cfg::node
 		cfg::_enum<CellSysutilLang> language{ this, "Language", CellSysutilLang{1} }; // CELL_SYSUTIL_LANG_ENGLISH_US
 		cfg::_enum<CellKbMappingType> keyboard_type{ this, "Keyboard Type", CellKbMappingType{0} }; // CELL_KB_MAPPING_101 = US
 		cfg::_enum<enter_button_assign> enter_button_assignment{ this, "Enter button assignment", enter_button_assign::cross };
+		cfg::_int<std::numeric_limits<s64>::min(), std::numeric_limits<s64>::max()> console_time_offset{ this, "Console time offset (s)", 0 };
```

Comment:
Maybe add a sane limitation here like -+ 100 years (60 * 60 * 24 * 365 * 10 seconds). Its bad to allow users to set huge values which rpcs3 won't handle correct anyways and I think by the time being users won't need more than that.

Reply:
I concur.  Have pushed the time recommended (+/-100years [60*60*25*365*100]).